### PR TITLE
css unit testing

### PR DESCRIPTION
**Pull request by JuanCaicedo**

### CODE REVIEW

File: css-unit-testing.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+[Test-Driven Development By Example](https://www.amazon.com/Test-Driven-Development-Kent-Beck/dp/0321146530),
+by programming legend Kent Beck.
+
+That book really convinced me of the long term benefits of testing for writing
+maintainable code. For me the benefit of tests (and especially test driven
+development) is that they give you a safe way to refactor your code.
+
+## Refactoring, with tests
+
+All systems trends towards complexity. In other words, overtime all projects
```
Comment (danpaz) [inline css-unit-testing.md]: typo: `systems trend`

Comment (danpaz) [reply css-unit-testing.md]: typo: `over time`

File: css-unit-testing.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+I find refactoring to be one of the most valuable things a software engineer can
+do because it allows you to develop future features in that codebase much more easily.
+
+However, refactoring naively has a high risk of introducing bugs into your code.
+
+Building up good test coverage gives you a quick way of verifying that all your
+code works the way you expect it to. That way after you refactor code, you can
+be sure that you haven't changed its behavior.
+
+## How I got to front-end development
```
Comment (danpaz) [inline css-unit-testing.md]: `into`?

File: css-unit-testing.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+Sounds good to me!
+
+## Where did all my tests go?
+
+As I started reading through our `styles/` directory and getting more familiar
+with all our CSS code, I realized something surprising.
+
+A lot of it was kind of a mess.
+
+Not for any particular reason. Overtime requirements have changed, as has the
```
Comment (danpaz) [inline css-unit-testing.md]: `Over time`

File: css-unit-testing.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+structure of the site and our knowledge of best practices. Also, the browsers we
+support have changed as, have the tricks we have to do to make the site
```
Comment (danpaz) [inline css-unit-testing.md]: `have changed, as have`

**Review by danpaz — COMMENTED** [review]
Nice post! Just pointed out some typos.

**Comment by JuanCaicedo** [comment]
Thanks @danpaz !

**Pull request merged**